### PR TITLE
[tflchef] Add BidirectionalSequenceLSTM Operation

### PR DESCRIPTION
**Pull request by dr-venkman**
This commit add BidirectionalSequenceLSTM operation on TFlchef.

ONE-DCO-1.0-Signed-off-by: venkat.iyer venkat.iyer@samsung.com

### CODE REVIEW

**Review by mhs4670go — DISMISSED** [review]
LGTM

File: compiler/tflchef/proto/tflchef.proto [hunk 1]
```diff
@@ -110,6 +110,15 @@ enum MirrorPadMode {
   SYMMETRIC = 1;
 }
 
+message BidirectionalSequenceLSTMOptions {
+  optional Activation activation = 1 [default = NONE];
+  optional float cell_clip = 2 [default = 0.0];
+  optional float proj_clip = 3 [default = 0.0];
+  optional bool time_major = 4 [default = false];
+  optional bool asymmetric_quantize_inputs = 5 [default = false];
+  optional bool merge_outputs = 6 [default = false];
```
Comment (struss) [inline compiler/tflchef/proto/tflchef.proto:119]: Is it okay that the order of option is not matching with TFlite `schema.fbs` order?
and Default value is little bit different on `time_major`
(this comment referenced from [here](https://github.com/tensorflow/tensorflow/blob/f3c5b4ea29b1bd536bd70fc332cc1ccdf1b09695/tensorflow/lite/schema/schema.fbs#L637-L654))

Comment (mhs4670go) [reply compiler/tflchef/proto/tflchef.proto:119]: @dr-venkman Seems technically okay. But we might as well follow the order of tflite schema:)

Comment (dr-venkman) [reply compiler/tflchef/proto/tflchef.proto:119]: @struss, @mhs4670go , thanks for the comment. I will be correcting the order in the next commit.

**Review by mhs4670go — APPROVED** [review]
LGTM

**Review by struss — APPROVED** [review]
LGTM! Thanks!

**Pull request merged**